x/stake/genesis.go

```diff
@@ -21,8 +21,8 @@ func InitGenesis(ctx sdk.Context, keeper Keeper, data types.GenesisState) (res [
 
 	// We need to pretend to be "n blocks before genesis", where "n" is the validator update delay,
 	// so that e.g. slashing periods are correctly initialized for the validator set
-	// e.g. with a one-block offset - the first TM block is at height 0, so state updates applied from genesis.json are in block -1.
-	ctx = ctx.WithBlockHeight(-types.ValidatorUpdateDelay)
+	// e.g. with a one-block offset - the first TM block is at height 1, so state updates applied from genesis.json are in block 0.
```

Comment:
Tendermint used to start at block height 0, now it starts at 1, so we needed to update this.